tools/openms/IDFileConverter.xml

```diff
@@ -69,6 +70,79 @@ python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hard
       <option value="score">score</option>
       <expand macro="list_string_san" name="score_type"/>
     </param>
+    <section name="fasta_to_mzml" title="[FASTA input + MzML output only] Parameters used to adjust simulation of the theoretical spectra" help="" expanded="false">
```

Comment:
A prime example why hand-written wrappers are much more userfriendly.

Reply:
I agree that this is not ideal .. but I'm wondering how this should look like more user friendly? A conditional over all combinations of input and output type?